hw/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import './Button.css'
+
+type ButtonProps = {
+    children: React.ReactNode;
+    onClick: (e: React.MouseEvent) => void;
+    disabled?: any
```

Comment:
это `boolean` значение
```suggestion
    disabled?: boolean;
```